docs/library/utime.rst

```diff
@@ -28,8 +28,8 @@ by following means:
   in such case).
 
 If actual calendar time is not maintained with a system/MicroPython RTC,
-functions below which require reference to current absolute time may
-behave not as expected.
+functions below which require reference to current absolute time may not
```

Comment:
You're of course using "may not" in the sense "not allowed to", as in "You may not make changes for bad" ? If so, your change is incorrect. And if you don't use "may not" in this sense, then why do you make a change which replaces the unambiguous phrase with an ambiguous one, which will cause confusion?